mergify_engine/worker.py

```diff
@@ -315,6 +315,7 @@ async def consume(self, stream_name: StreamNameType) -> None:
                         owner_id, owner_login, sub, client, redis_cache
                     )
                     pulls = await self._extract_pulls_from_stream(installation)
+                    client.set_requests_ratio(len(pulls))
```

Comment:
can this be 0 ? 🤔

Reply:
no, in theory, since we can't be here without pulls, I will ensure we don't divide by 0